src/main/kotlin/community/flock/eco/workday/utils/DateUtils.kt

```diff
@@ -47,7 +47,7 @@ object DateUtils {
         return countWorkDaysInPeriod(from, to)
     }
 
-    val humanReadableDateFormat = DateTimeFormatter.ofPattern("dd-MM-yyyy")
+    val humanReadableDateFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy")
```

Comment:
Uit nieuwsgierigheid: waarom?

Reply:
Tip van intellij, omdat je een platformcall doet kan hij niet garanderen dat dit wel of niet nullable is.
![image](https://github.com/user-attachments/assets/f6826547-f19d-4050-969d-26653afd1b53)